html/install.php

```diff
@@ -39,7 +39,7 @@
 $config['db_socket']=$dbsocket;
 
 if (!empty($config['db_socket'])) {
-    $config['db_host'] = '';
```

Comment:
This bug is actually in dbConnect in my opinion.

Inserting this in line 60 of dbFacile.php should fix it without breaking anyone.
```
    if (empty($host)) {
        $host = 'localhost';  //needed because of persistent p: below
    }
```